src/loaders/OMETiffLoader.js

```diff
@@ -95,7 +96,15 @@ export default class OMETiffLoader {
       this.omexml.getNumberOfImages() || (SubIFDs && SubIFDs.length);
     this.isBioFormats6Pyramid = SubIFDs;
     this.isPyramid = this.numLevels > 1;
-
+    // The omexml specification only allows for these - zarr is more flexible so this
+    // is for unifying the two loaders in upstream applications.
+    this.dimensions = [
+      { field: 'channel', type: 'nominal', values: this.channelNames },
+      { field: 'z', type: 'number', values: null },
+      { field: 'time', type: 'number', values: null },
+      { field: 'x', type: 'quantitative', values: null },
+      { field: 'y', type: 'quantitative', values: null }
```

Comment:
The reason that `x` and `y` are `null` for values is that these dimensions will not have selectors in the UI, and I didn't want to include long arrays of unused indices. Other dimensions we will need to communicate what selections are available. Ideally attributes that describe a dimension should be on these objects, and adopting something similar to [x-array](http://xarray.pydata.org/en/stable/) model would be great. 

The reason I check for `ordinal` and `nominal` is that you can simply perform an `array.indexOf()` to get the right index. Perhaps the vega-lite stuff doesn't really work for our use case and something like:

```javascript
[
      { field: 'channel', type: 'nominal', values: this.channelNames },
      { field: 'z', type: 'physicalSize', unit: 'uM', value: 10, maxIndex: 4 },
      { field: 'time', type: 'temporal', unit: 'min', values: [2, 3, 4, 5, 10] },
      { field: 'x', type: 'physicalSize', unit: 'uM', value: 1, maxIndex: 50000 },
      { field: 'y', type: 'physicalSize', unit: 'uM', value: 1, maxIndex: 40000 }
]
```
would be better.

Reply:
I don't think that z-stack metadata is communicated normally.  I think we'll do `ordinal` for now and just provide the range.